templates/glance/config/00-config.conf

```diff
@@ -79,7 +79,11 @@ enable_proxy_headers_parsing=True
 
 [paste_deploy]
 config_file = /etc/glance/glance-api-paste.ini
```

Comment:
@konan-abhi I'm wondering if L81 can be removed: all the relevant glance related config files are located in `/etc/glance/glance.conf.d` and that `glance-api-paste.ini` comes with the `Glance` container.
What do you think about removing L81?

Reply:
not required, we can remove it.

Reply:
Sorry I'm probably wrong there, ignore my previous comment.

Reply:
Done